pkg/configuration/cache_test.go

```diff
@@ -343,11 +343,11 @@ func TestMultipleExecutionsInParallel(t *testing.T) {
 			config, secrets, err := GetConfig(cl, &toolchainv1alpha1.ToolchainConfig{})
 
 			// then
-			require.NoError(t, err)
+			assert.NoError(t, err)
 			toolchaincfg, ok := config.(*toolchainv1alpha1.ToolchainConfig)
-			require.True(t, ok)
+			assert.True(t, ok)
 			assert.NotEmpty(t, toolchaincfg.Spec)
-			require.NotEmpty(t, secrets)
+			assert.NotEmpty(t, secrets)
```

Comment:
is there any reason for changing it to `assert`? IIRC, the best practice is to use `require` for some specific assertions like the error one.

Reply:
you're right,
linter check in host suggested to use require instead of assert.
but I was changing it to assert here because of linter complaining previously - in this commit https://github.com/codeready-toolchain/toolchain-common/pull/376/commits/0986552a344431731de2d9c8cf0c3484a60c5553

Let me check what's going on. I expect the linter to complain uniformly across all repos.  but in general i do agree require is the convention to usr instead of assert with error especially.

Reply:
its because of `"go-require: require must only be used in the goroutine running the test function (testifylint)"` added a comment beside it for explanation

Reply:
ah, ok. This is a bit different since we run the verification inside our goroutine :+1: 
Thanks for taking a look at it